src/python/examples/simple_grpc_infer_client.py

```diff
@@ -96,7 +96,12 @@
             ssl=FLAGS.ssl,
             root_certificates=FLAGS.root_certificates,
             private_key=FLAGS.private_key,
-            certificate_chain=FLAGS.certificate_chain)
+            certificate_chain=FLAGS.certificate_chain,
+            grpc_arg_keepalive_time_ms=2**31-1,
```

Comment:
Expose these parameters as the script arguments. Ensure you document the whole thing well. 

These defaults can be easily set in the parser.  See L36-L89.